[cli] Cannot find module '../lib/undefined'

https://github.com/mapbox/ecs-watchbot/blob/d1dda99b07736431365bdc58116374206cbd0687/bin/cli.js#L26-L28

If the user does not specify any arguments and just calls `watchbot`, an error message is printed

```
Cannot find module '../lib/undefined'
```

The code here should be adjusted to provide a message that makes it clear you **must** provide a first argument. 

Furthermore, for `dead-letter` and `worker-capacity` commands, the caller **must** provide `-r` and `-s` flag values, but there's no validation of the user input, which leads to confusing failures. 

cc @dputtick

--- a/bin/cli.js
+++ b/bin/cli.js
@@ -24,10 +24,15 @@ var cli = meow({
 });
 
 var command = cli.input[0];
+if (!command) {
+  console.error('ERROR: You must specify a command');
+  cli.showHelp(1);
+}
+
 var fn;
 try { fn = require(`../lib/${command}`); }
 catch(err) {
-  console.error(err.message);
+  console.error('"' + command + '" is not a valid command');
   cli.showHelp(1);
 }
 

